src/legacy/ui/public/courier/courier.js

```diff
@@ -62,29 +57,12 @@ uiModules.get('kibana/courier').service('courier', ($rootScope, Private) => {
         // resume polling.
         searchPoll.pause();
 
-        // And abort all pending requests.
-        searchRequestQueue.abortAll();
```

Comment:
Individual calls to `searchSource.fetch()` are now responsible for aborting those requests when they see fit.